tests/bin/eslint.js

```diff
@@ -127,6 +127,22 @@ describe("bin/eslint.js", () => {
             return Promise.all([exitCodeAssertion, outputFileAssertion]);
         });
 
+        it("lets you know that files were fixed", () => {
+            const child = runESLint(["--fix", "--no-eslintrc", "--no-ignore", tempFilePath]);
+
+            return getStdout(child).then(stdout => {
+                assert.match(stdout, /Fixes were applied/);
+            });
+        });
+
+        it("won't falsely tell you that files were fixed", () => {
+            const child = runESLint(["--fix", "--no-eslintrc", "--no-ignore", expectedFixedText]);
```

Comment:
Good catch, you are correct. this test has been removed in the most recent iteration, however